index.js

```diff
@@ -46,10 +54,11 @@ FeatureParser.prototype.getTestFrameworkImport = function() {
     case 'mocha':
       return 'import { afterEach, beforeEach, describe, it } from \'mocha\';';
     default: // qunit
-      return 'import { module, test } from \'qunit\';';
+      return 'import { moduleFor, moduleForComponent, test } from \'ember-qunit\';\
+              import {module} from \'qunit\';';
```

Comment:
please update the whitespace for this line to `import { module } from \'qunit\'`.